spidermon/contrib/actions/email/__init__.py

```diff
@@ -86,12 +86,10 @@ def from_crawler_kwargs(cls, crawler):
 
     def run_action(self):
         message = self.get_message()
-        if not self.fake:
-            self.send_message(message)
+        if self.fake:
+            logging.info(message)
         else:
-            print("-" * 40)
-            print(message.as_string())
-            print("-" * 40)
+            self.send_message(message)
```

Comment:
`message` is a `email.mime.multipart.MIMEMultipart` object, not a string.
Use `message.as_string()` instead of just `message` in the logs.

Reply:
Done!